src/NUnitFramework/framework/TestContext.cs

```diff
@@ -240,85 +173,12 @@ public static void Write(string format, params object?[] args)
             Out.Write(format, args);
         }
 
-        /// <summary>Write a line terminator to the current result</summary>
-        public static void WriteLine()
```

Comment:
@OsirisTerje I'm not 100% sure as I'm on my phone at the moment, but wouldn't this one still be needed?

Reply:
Actually, even if the other code is _compile_ time compatible, it still isn't binary compatible.
If someone has a NUnitExtension library calling the old methods, those references no longer can be resolved.
This can only be done in a major version and as per my other comment generics here doesn't work as the code called is not generic.